lua/nordic/colors/init.lua

```diff
@@ -96,4 +96,7 @@ function C.build_palette()
     C.comment = C.gray4
 end
 
+-- Build the first palette.
+C.build_palette()
```

Comment:
This does not have to be here.

Reply:
It turns out that is necessary if the user imports `nordic.colors` in their config (before calling setup)...

So I figured I would just sneak it in this PR.

Reply:
They probably shouldn't do that, but yeah let's leave it in.

Reply:
I will actually remove that part from the readme now that I think about it.